agrirouter-sdk-java-api/src/main/kotlin/com/dke/data/agrirouter/api/service/parameters/base/AbstractParameterBase.kt

```diff
@@ -1,11 +1,19 @@
 package com.dke.data.agrirouter.api.service.parameters.base
 
-open class AbstractParameterBase : DynamicAttributesStorage() {
+abstract class AbstractParameterBase : DynamicAttributesStorage() {
 
-    var applicationMessageId : String? = null
+    var applicationMessageId: String? = null
 
-    var teamsetContextId : String? = null
+    var teamsetContextId: String? = null
 
-    var sequenceNumber : Int = 1
+    var sequenceNumber: Int = 1
+
+    /**
+     * Trim the parameters to remove whitespaces and avoid problems.
+     */
+    fun trimParameterBase() {
+        applicationMessageId = applicationMessageId?.trim()
+        teamsetContextId = teamsetContextId?.trim()
```

Comment:
So far we have no definition in the agrirouter documentation that disallows spaces in the TeamsetContextID. There is a certfication test case though to make sure that applicationMessageIds are a UUID. 

I see that it makes sense to have one but currently there is a (very very low) chance of breaking  an app that could work with spaces in the teamsetContextId.

Reply:
Since the method was not called, this should not be a blocker. The internal discussion did not deliver any result, therefore I removed the method.